.env.example

```
@@ -17,3 +17,6 @@ MAILER_SECURE=false # should be either 'true' or anything else (evaluates to fal
 MAILER_USER=api
 MAILER_PASSWORD=api
 MAILER_FROM=no-reply@paastech.cloud
+
+# Seed data for development
+DEV_USER_PASSWORD=PaastechDev1!
```

I suggest adding a way to load the public ssh key from a given location, so it can get easier to run.

```suggestion
DEV_USER_PASSWORD=PaastechDev1!
DEV_USER_SSH_PUBKEY_FILE=/my/dev/ssh/keyfile
```